lib/github_adapter.rb

```diff
@@ -3,15 +3,27 @@
 class GithubAdapter
   cattr_accessor :user, :password
 
-  def resource
-    @resource ||= RestClient::Resource.new('https://api.github.com/orgs/chatto-hub-test2/teams', user, password)
+  def create_team_resource
+    @resource ||= RestClient::Resource.new(
+      'https://api.github.com/orgs/chatto-hub-test2/teams',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.